src/runtime/bootstrap-lazy.ts

```diff
@@ -49,6 +50,13 @@ export const bootstrapLazy = (lazyBundles: d.LazyBundlesRuntimeData, options: d.
     // async queue. This will improve the first input delay
     plt.$flags$ |= PLATFORM_FLAGS.appLoaded;
   }
+  if (BUILD.hydrateClientSide && BUILD.shadowDom) {
+    const styles = doc.querySelectorAll(`[${HYDRATED_STYLE_ID}]`);
+    let i = 0;
+    for (; i < styles.length; i++) {
+      registerStyle(styles[i].getAttribute(HYDRATED_STYLE_ID), convertScopedToShadow(styles[i].innerHTML), true);
+    }
```

Comment:
This seems to be the same as in `src/runtime/bootstrap-custom-element.ts`, worth wrapping it into a utility function?